### PR TITLE
update pt-br for #1582

### PR DESCRIPTION
**Pull request by netrik182**
Fixed one typo from last PR.

### CODE REVIEW

File: translations/monero-core_pt-br.ts [hunk 1]
```diff
@@ -1037,37 +1037,37 @@
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="60"/>
         <source>GUI version: </source>
-        <translation type="unfinished"></translation>
+        <translation>Versão da Interface</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-br.ts]: missing `:` + whistespace

File: translations/monero-core_pt-br.ts [hunk 2]
```diff
@@ -149,42 +149,42 @@
...
         <location filename="../pages/History.qml" line="237"/>
         <source>Date</source>
-        <translation type="unfinished">Data</translation>
+        <translation>Data</translation>
     </message>
     <message>
         <location filename="../pages/History.qml" line="278"/>
         <source>No history...</source>
-        <translation type="unfinished"></translation>
+        <translation>Sem histórico</translation>
```
Comment (netrik182) [inline translations/monero-core_pt-br.ts]: Missing `...`

File: translations/monero-core_pt-br.ts (hunk 1 above)
Comment (netrik182) [reply translations/monero-core_pt-br.ts]: Thank you. If @vp1111 could review the file today, I send the PR again tonight (about 10:00pm UTC).

File: translations/monero-core_pt-br.ts [hunk 3]
```diff
@@ -1079,235 +1079,242 @@ The following information will be deleted
...
         <location filename="../pages/settings/SettingsWallet.qml" line="262"/>
         <source>Show seed &amp; keys</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar semente &amp; chaves</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsWallet.qml" line="276"/>
         <source>Store this information safely to recover your wallet in the future.</source>
-        <translation type="unfinished"></translation>
+        <translation>Armazenar essas informações com segurança para recuperar sua carteira no futuro.</translation>
```
Comment (netrik182) [inline translations/monero-core_pt-br.ts]: Use the word `Armazena` instead of `Armazenar`

File: translations/monero-core_pt-br.ts [hunk 4]
```diff
@@ -1403,7 +1410,7 @@ The old wallet cache file will be renamed and can be restored later.
...
-        <translation type="unfinished"></translation>
+        <translation>Para evitar anular a proteção oferecida pelas assinaturas de anel do Monero, uma saída não deve ser gasta com anéis diferentes em blockchains diferentes. Embora isso normalmente não seja uma preocupação, ela pode se tornar uma quando um clone do Monero que reutiliza chaves permita que você gaste as saídas existentes. Nesse caso, você precisa garantir que essas saídas existentes usem o mesmo anel em ambos blockchains.&lt;br&gt;Isso será feito automaticamente pelo Monero e qualquer software de reutilização de chaves que não esteja tentando privá-lo ativamente de sua privacidade.&lt;br&gt;Se você estiver usando um clone do Monero que também reutiliza chaves, e esse clone não incluir essa proteção, você ainda poderá garantir que suas transações sejam protegidas por gastos primeiramente no clone e, em seguida, adicionar manualmente o anel nesta página, o que lhe permite gastar seu Monero com segurança.&lt;br&gt;Se você não usar um clone do Monero que reutiliza chaves sem esses recursos de segurança, não será necessário fazer nada, pois tudo é automatizado.&lt;br&gt;</translation>
```
Comment (netrik182) [inline translations/monero-core_pt-br.ts]: Use `[...] ao gastar primeiramente no clone [...]`

File: translations/monero-core_pt-br.ts (hunk 1 above)
Comment (erciccione) [reply translations/monero-core_pt-br.ts]: No need to send another PR, you can just push the new commits on this one

**Comment by erciccione** [comment]
@netrik182 Could you squash into just one commit?

**Pull request merged**